core/jazz_ui/e2e/04-jazzservices/jazzservices.e2e-spec.ts

```diff
@@ -153,43 +153,66 @@ describe('Overview', () => {
             jazzServices_po.getTestAPI().click();
             browser.sleep(10000);
             browser.getAllWindowHandles().then(function(handles){
-                  browser.switchTo().window(handles[1]).then(function(){
-                        jazzServices_po.getAPIGET().click();
-                        browser.sleep(5000);
-                        jazzServices_po.getTryOut().click();
-                        browser.sleep(5000);
-                        jazzServices_po.getStringA().sendKeys('Testing');
-                        jazzServices_po.getStringB().sendKeys('Jazz');
-                        browser.sleep(5000);
-                        jazzServices_po.getExecute().click();
-                        jazzServices_po.getAPIGET().click();
-                        browser.close();
-                  });
-                  browser.switchTo().window(handles[0]).then(function(){
-                        browser.sleep(15000);
-                        browser.wait(EC.visibilityOf(jazzServices_po.getMetrices()), timeOutHigh);
-                        jazzServices_po.getMetrices().click();
-                        browser.sleep(15000);
-                        jazzServices_po.getMetricsChildOne().click();
-                        browser.sleep(4000);
-                        jazzServices_po.getMetricsChildTwo().click();
-                        browser.sleep(4000);
-                        jazzServices_po.getMetricsChildThree().click();
-                        browser.sleep(4000);
-                        jazzServices_po.getMetricsChildFour().click();
-                        browser.sleep(4000);
-                        jazzServices_po.getMetricsChildFive().click();
-                        browser.sleep(4000);
-                        jazzServices_po.getMetricsChildSix().click();
-                        browser.sleep(4000);
-                        jazzServices_po.getMetricsChildSeven().click();
-                        browser.sleep(4000);
+                browser.switchTo().window(handles[1]).then(function(){
+                    jazzServices_po.getAPIGET().click();
+                    browser.sleep(5000);
```

Comment:
are these sleep  of 5 secs required in line number 158, 160, and 163?

Reply:
removed